core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1144,12 +1144,17 @@ private function encodeHtmlAttributes($html)
 				. ')?+'                   // Assignment is optional
 			. ')*+'                       // Attributes may occur zero or more times
 			. '[\s/]*+'                   // Optional white space including slash
-			. '>?'                        // Tag end (optional if EOF)
+			. '>?+'                       // Tag end (optional if EOF)
 			. '|<!--'                     // Or comment
 			. '|<!'                       // Or bogus ! comment
 			. '|<\?'                      // Or bogus ? comment
 			. '|</(?![a-z])'              // Or bogus / comment
-		. ')i';
+		. ')iS';
+
+		if (strpos($html, '{{') === false && strpos($html, '}}') === false)
+		{
+			return $html;
+		}
```

Comment:
Should we move the early return above the `$tagRegEx` definition?

Reply:
Done in b0d740e0d6306bc2ac838ab32108a8add934ac64